manta-js/package/package.json

```diff
@@ -15,7 +16,10 @@
     "wasm-postbuild": "rm ./dist/wallet/crate/pkg/.gitignore",
     "wasm-dev": "wasm-pack build ./src/wallet/crate",
     "wasm": "wasm-pack build ./src/wallet/crate --out-dir ../../../dist/wallet/crate/pkg",
-    "build": "yarn prebuild && yarn wasm && yarn wasm-dev && tsc && yarn wasm-postbuild"
+    "build": "yarn prebuild && yarn wasm && yarn wasm-dev && tsc && yarn wasm-postbuild",
+    "wasm-dev-node": "wasm-pack build --target nodejs ./src/wallet/crate",
```

Comment:
If we want to support node I think we would have to publish a node npm package as well, which is a bit annoying

Reply:
Can we support it from 1 branch or we'll need multiple branches as well?

Reply:
So are more changes needed or is this pretty much the final version ?

Reply:
No this is it, other than writing docs. 

This should work without creating another branch. For browser run `yarn build`  and for node `yarn build-node`. This works for both for me, ran the example to (lightly) test for browser and tested the node build on local project https://github.com/ferrell-code/node-manta-example

Reply:
added some simple docs in README

Reply:
Ok thanks, let me check out the node version with my test in manta.

Reply:
Nice, seems im finally hitting an internal SDK error:
```
Initial sync failed. Error during asynchronous call: SignerConnectionError(reqwest::Error { kind: Builder, source: "JsValue(ReferenceError: Headers is not defined\nReferenceError: Headers is not defined\n    at /home/georgi/Desktop/workspace/Manta/tests/node_modules/manta.js/dist/wallet/crate/pkg/manta_wasm_wallet.js:1219:17\n    at handleError (/home/georgi/Desktop/workspace/Manta/tests/node_modules/manta.js/dist/wallet/crate/pkg/manta_wasm_wallet.js:251:18)\n    at module.exports.__wbg_new_2d0053ee81e4dd2a (/home/georgi/Desktop/workspace/Manta/tests/node_modules/manta.js/dist/wallet/crate/pkg/manta_wasm_wallet.js:1218:65)\n    at wasm://wasm/0036a466:wasm-function[936]:0x9bfa8\n    at wasm://wasm/0036a466:wasm-function[64]:0x134e5\n    at wasm://wasm/0036a466:wasm-function[175]:0x56ed4\n    at wasm://wasm/0036a466:wasm-function[81]:0x248e4\n    at wasm://wasm/0036a466:wasm-function[184]:0x59e8a\n    at wasm://wasm/0036a466:wasm-function[182]:0x5933e\n    at wasm://wasm/0036a466:wasm-function[686]:0x942f3\n    at wasm://wasm/0036a466:wasm-function[411]:0x7fa62\n    at wasm://wasm/0036a466:wasm-function[1045]:0x9d39f\n    at __wbg_adapter_28 (/home/georgi/Desktop/workspace/Manta/tests/node_modules/manta.js/dist/wallet/crate/pkg/manta_wasm_wallet.js:210:10)\n    at real (/home/georgi/Desktop/workspace/Manta/tests/node_modules/manta.js/dist/wallet/crate/pkg/manta_wasm_wallet.js:195:20)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5))" })
```

Reply:
Tried it and it works.